package.json

```diff
@@ -5,6 +5,11 @@
 	"license": "MIT",
 	"repository": "sindresorhus/ky",
 	"funding": "https://github.com/sindresorhus/ky?sponsor=1",
+	"main": "./umd.js",
+	"module": "./index.js",
+	"browser": "./index.js",
+	"unpkg": "./index.js",
+	"types": "./index.d.ts",
```

Comment:
This is not needed. TS infers this automatically when named `index.d.ts`.

Reply:
Ok, I thought that explicit it better than implicit